test/MarketMaker.js

```diff
@@ -344,45 +328,37 @@ contract('marketMaker', function(accounts) {
     }
     assert.equal(true, thrown, "claimEth should have thrown");
 
-    try {
-      thrown = false;
-      await claimEth(0.03);
-    } catch(e) {
-      thrown = true;
-    }
-    assert.equal(true, thrown, "claimEth should have thrown");
-
-    // Claim 0.012 eth
-    await claimEth(0.012);
+    // Claim some ether
+    let toClaim = (await getMaxClaimableWei(state)).dividedBy(2);
```

Comment:
can we use `state.marketMakerMaxClaimableWei` here, not divided by 2? We would be claiming 100% of the claimable at this point, it's a nice symmetry to previous assertion that fails with that + 1

Reply:
:+1: